tests/telemetry_and_hold_test.py

```
@@ -0,0 +1,268 @@
+
+        # Await the actual main scene initialization log (signaling swap completed)
+        wait_for_console_log(
+            logs,
+            lambda text: "initializing main scene" in str(text).lower(),
+            start_click_idx,
+            page,
+            timeout_ms=TEST_TIMEOUT,
+        )
+
+        # Locate exact instances of the log dictionaries to compute Delta T
+        load_log = next((l for l in logs[start_click_idx:] if "scene loaded successfully" in str(l["text"]).lower()), None)
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)